File: TypeScript/2390. Removing Stars From a String.ts
// noinspection JSUnusedGlobalSymbols
const removeStars = (s: string): string => {
    const arr: string[] = []

    for (let i = 0; i < s.length; i++) {
        const word = s[i]
        if (word === '*') {
            arr.pop()
        } else {
            arr.push(word)
        }
    }

    return arr.join('')
}

File: Python/2390. Removing Stars From a String.py
# noinspection PyMethodMayBeStatic
class Solution:
    def removeStars(self, s: str) -> str:
        array = []

        for word in list(s):
            if word == '*':
                array.pop()
            else:
                array.append(word)

        return ''.join(array)
